students/grzegorz_bajorski/lesson_04_unit_testing/the_length_of_the_segment_test.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from the_length_of_the_segment import distance
+
+
+class MyTest(pytest):
```

Comment:
To group, multiple tests class name should start from ```Test``` word. So this line should look:
```class TestMy(object)```

Reply:
Not all of the tests are passed, please fix them